src/wpsmon.c

```diff
@@ -52,7 +52,6 @@ static struct mac {
 	unsigned char vendor_oui[1+3];
 	unsigned char probes;
 	unsigned char flags;
-	uint32_t checksum;
```

Comment:
if we don't need checksum anymore, we should also remove it from libwps.c/h

Reply:
basically revert the changes i did to libwps.c related to checksum, including inclusion of crc.h